### PR TITLE
Webp announcement blog and changelog

### PR DESCRIPTION
**Pull request by SnezhannaM**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/blog/post/webp-support-for-safari/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+layout: post
+title: "WebP support now available for Safari on all devices"
+description: With WebP now supported in Safari, Appwrite developers can use this format on any browser and device.
+date: 2024-10-02
+cover: /images/blog/webp-support-for-safari/cover.png
+timeToRead: 3
+author: eldad-fux
+category: product
+---
+We’re happy to share that Appwrite Cloud users now has full WebP support across all versions of Safari browsers and devices.
```
Comment (ItzNotABug) [inline src/routes/blog/post/webp-support-for-safari/+page.markdoc:11]:
```suggestion
We’re happy to share that Appwrite Cloud users now have full WebP support across all versions of Safari browsers and devices.
```

**Review by eldadfux — APPROVED** [review]

File: src/routes/blog/post/webp-support-for-safari/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+However, as outlined on [Can I use WebP?](https://caniuse.com/webp), Safari has supported WebP for quite some time now. With this update, Appwrite has fully embraced WebP across all Safari browsers and devices, eliminating the need for fallback solutions. This update is now available to all Appwrite Cloud users, making image delivery more efficient across all platforms.
+
+# Why WebP matters
+
+WebP is a modern image format offering superior compression while maintaining high image quality. Compared to traditional formats like JPEG and PNG, WebP can reduce image sizes by up to 30%, resulting in faster page load times, lower bandwidth usage, and an overall better user experience. This is especially critical for developers looking to optimize their applications for performance and scalability.
+
+With WebP now fully supported in Safari, Appwrite developers can take full advantage of this format across all browsers and devices. This will lead to reduced latency, less bandwidth consumption, and cost savings - particularly for Cloud accounts, where efficiency matters most.
```
Comment (ItzNotABug) [inline src/routes/blog/post/webp-support-for-safari/+page.markdoc:21]:
```suggestion
With WebP now fully supported in Safari, Appwrite developers can take full advantage of this format across all browsers and devices. This improvement will lead to reduced latency, less bandwidth consumption, and cost savings - particularly for Cloud accounts, where efficiency matters most.
```

File: src/routes/changelog/(entries)/2024-10-02.markdoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+layout: changelog
+title: "WebP support now available for Safari on all devices"
+date: 2024-10-02
+cover: /images/changelog/2024-10-02.png
+---
+
+We’re happy to share that Appwrite Cloud users now has full WebP support across all versions of Safari browsers and devices.
```
Comment (ItzNotABug) [inline src/routes/changelog/(entries)/2024-10-02.markdoc:8]:
```suggestion
We’re happy to share that Appwrite Cloud users now have full WebP support across all versions of Safari browsers and devices.
```

File: src/routes/blog/post/webp-support-for-safari/+page.markdoc (hunk 1 above)
Comment (Herry0092) [reply src/routes/blog/post/webp-support-for-safari/+page.markdoc]: Good to share with you 😄

**Pull request merged**